protocol/communities/manager.go

```diff
@@ -1417,6 +1417,11 @@ func (m *Manager) handleCommunityDescriptionMessageCommon(community *Community,
 	}
 	community.config.EventsData = nil
 
+	// Set Joined if we are part of the member list
+	if !community.Joined() && community.hasMember(&m.identity.PublicKey) {
```

Comment:
If one of the two conditions doesn't work here, I think the problem may be elsewhere

Reply:
Are you saying I should do this elsewhere? I don't fully get your point